test/parallel/test-tls-getcipher.js

```diff
@@ -55,3 +55,27 @@ server.listen(0, '127.0.0.1', common.mustCall(function() {
     server.close();
   }));
 }));
+
+if (!require('constants').TLS1_3_VERSION)
+  return console.log('cannot test TLSv1.3 against 1.3-incapable shared lib');
+
+tls.createServer({
+  key: fixtures.readKey('agent2-key.pem'),
+  cert: fixtures.readKey('agent2-cert.pem'),
+  ciphers: 'TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_CCM_8_SHA256',
+  maxVersion: 'TLSv1.3',
```

Comment:
dangling comma

Reply:
this is the same as master, see previous.